src/ol/format/KML.js

```diff
@@ -888,6 +903,74 @@ class KML extends XMLFeature {
     return regions;
   }
 
+  /**
+   * Read the cameras of the KML.
+   *
+   * @param {Document|Element|string} source Source.
+   * @return {Array<Object>} Cameras.
+   * @api
+   */
+  readCamera(source) {
+    const cameras = [];
+    if (typeof source === 'string') {
+      const doc = parse(source);
+      extend(cameras, this.readCameraFromDocument(doc));
+    } else if (isDocument(source)) {
+      extend(
+        cameras,
+        this.readCameraFromDocument(/** @type {Document} */ (source))
+      );
+    } else {
+      extend(cameras, this.readCameraFromNode(/** @type {Element} */ (source)));
+    }
+    return cameras;
+  }
+
+  /**
+   * @param {Document} doc Document.
+   * @return {Array<Object>} Cameras.
+   */
+  readCameraFromDocument(doc) {
+    const cameras = [];
+    for (let n = /** @type {Node} */ (doc.firstChild); n; n = n.nextSibling) {
+      if (n.nodeType === Node.ELEMENT_NODE) {
+        extend(cameras, this.readCameraFromNode(/** @type {Element} */ (n)));
+      }
+    }
+    return cameras;
+  }
+
+  /**
+   * @param {Element} node Node.
+   * @return {Array<Object>} Cameras.
+   * @api
+   */
+  readCameraFromNode(node) {
+    const cameras = [];
+    for (let n = node.firstElementChild; n; n = n.nextElementSibling) {
+      if (
+        includes(NAMESPACE_URIS, n.namespaceURI) &&
```

Comment:
```suggestion
        NAMESPACE_URIS.includes(n.namespaceURI) &&
```

Reply:
Thanks, for the hint. My fork was outdated, I synced the repo and updated the methods.